PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -396,6 +426,38 @@ fi
 
 # log_Msg "POSIXLY_CORRECT="${POSIXLY_CORRECT} #NOT DEFINED ANYWHERE ELSE DO WE NEED THIS? 
 
+if [ "$CustomBrain" = "ORIGMASK" ] ; then
+
+  RunMode=2
+
+  log_Msg "Skipping the step GradientNonlinearityAverage to ACPC Alignment applying custom mask in original space (<subject>/T1w/custom_mask.nii.gz). The custom mask is also used for brain extraction. This overrides the option of --runmode."
+  verbose_red_echo "---> Applying custom mask"
+
+  if [ "$(imtest ${T1wFolder}/custom_mask)" != 1 ] ; then
+       log_Err_Abort "ERROR: cannnot find ${T1wFolder}/custom_mask"
+  fi
+
+
+elif [[ "$CustomBrain" = "MASK" || "$CustomBrain" = "CUSTOM" ]] ; then
+
+  RunMode=5
+
+  if [ "$CustomBrain" = "MASK" ] ; then
+    log_Msg "Skipping all the steps to Atlas registration, applying custom mask in ACPC space (<subject>/T1w/custom_acpc_dc_restore_mask.nii.gz). This overrides the option of --runmode."
+    verbose_red_echo "---> Applying custom mask"
+
+    if [ "$(imtest ${T1wFolder}/custom_acpc_dc_restore_mask)" != 1 ] ; then
+         log_Err_Abort "ERROR: cannnot find ${T1wFolder}/custom_acpc_dc_restore_mask"
+    fi
+
+  # -- Then we are using existing images
+  else
+    log_Msg "Skipping all the steps preceding AtlasRegistration, using existing images instead. This overrides the option of --runmode."
```

Comment:
This else doesn't appear to override `RunMode`, though its message says it does.